text/0000-test-framework-binnacle.md

```diff
@@ -0,0 +1,357 @@
+- Start Date: 2020-01-07
+- Tracking: (leave this empty)
+- RFC PR: [kadalu/rfcs#0000](https://github.com/kadalu/rfcs/pull/0000)
```

Comment:
[rfcs#2](pull/6)

Reply:
depends on the order of merging when we have many rfcs :)

Reply:
chnaged to #6